curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-modern-javascript-methods-by-building-football-team-cards/63e95e39860dc5b01ebe9be0.md

```diff
@@ -452,7 +461,7 @@ const setPlayerCards = (arr = players) => {
 
   playerCards.innerHTML += arr.map(
     ({ name, position, number, isCaptain, nickname }) => {
-      `
+      return `
```

Comment:
that's not implicit return. Also I meant updating from step 24 (instructions), up to step 32